src/StimulusBundle/src/Dto/StimulusAttributes.php

```diff
@@ -218,6 +218,20 @@ private function normalizeControllerName(string $controllerName): string
         return preg_replace('/^@/', '', str_replace('_', '-', str_replace('/', '--', $controllerName)));
     }
 
+    /**
+     * Normalize a Stimulus controller's action event name.
+     *
+     * @see https://stimulus.hotwired.dev/reference/actions
+     */
+    private function normalizeEventName(?string $eventName): ?string
+    {
+        if (null === $eventName) {
+            return null;
+        }
+
```

Comment:
I suggest to not support nullable string here, but only non-nullable string.
```suggestion
    private function normalizeEventName(string $eventName): string
    {
```

The null check should be done before calling this method.

Reply:
Moved `null` value check out of the method.